tools/tm-bench/main.go

```diff
@@ -93,6 +94,9 @@ Examples:
 		"broadcast_tx_"+broadcastTxMethod,
 	)
 
+	//catch Interrupt and quit tm-bench
+	go catchInterrupt(transacters)
```

Comment:
I'd remove catchInterrupt altogether and just call

```go
go cmn.TrapSignal(func() {
  ...
}
```

why need in intermediate function?

Reply:
Thanks for your review! All was done according to your nice advice.